providers/os/connection/docker/container_connection.go

```diff
@@ -258,7 +258,7 @@ func NewContainerImageConnection(id uint32, conf *inventory.Config, asset *inven
 	}
 
 	// The requested image isn't locally available, but we can pull it from a remote registry.
-	if len(resolvedAssets) > 0 && resolvedAssets[0].Connections[0].Type == "container-registry" {
+	if len(resolvedAssets) > 0 && resolvedAssets[0].Connections[0].Type == string(shared.Type_RegistryImage) {
```

Comment:
maybe better to use `shared.Type_RegistryImage.String()` https://github.com/mondoohq/cnquery/blob/0dfbdd3682ff518f84b701f5f28e1ff03cbe4652/providers/os/connection/shared/shared.go#L22-L24

Reply:
oh, nice catch